### PR TITLE
#169817817 Implement requests table

### PR DESCRIPTION
**Pull request by VedaDom**
#### What does this PR do?
- The requests table is handy in displaying a users requests.

#### Description of Task to be completed?
- A logged in user should see the travel requests table along with the details of the requests that they have made.

#### How should this be manually tested?
After clone repository, go to the root of the project open path into command run the `npm install` and once setup is done click `npm start-dev`, this will start a local server which will then automatically open your default browser, then navigate to `http://localhost:8080/trips`.

#### What are the relevant pivotal tracker stories?
- [#169817817](https://www.pivotaltracker.com/story/show/169817817)

#### Screenshots (if appropriate)
Screen Shot 2020-02-24 at 12 35 29

Screen Shot 2020-02-21 at 02 55 27

Screen Shot 2020-02-24 at 14 33 00

Screen Shot 2020-02-24 at 14 35 17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,45 +65,47 @@
...
     "@fortawesome/free-brands-svg-icons": "^5.12.0",
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@material-ui/core": "^4.9.0",
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.41",
+    "@material-ui/pickers": "^3.2.10",
     "axios": "^0.19.2",
     "clean-webpack-plugin": "^2.0.0",
     "clsx": "^1.1.0",
-    "history": "^4.10.1",
```
Comment (Jajabenit250) [inline package.json:77]: kindly observe while rebasing accept current change first or both change but keep current change in their place. this line shouldn't have been changed.

File: package.json [hunk 2]
```diff
@@ -65,45 +65,47 @@
...
+    "moment": "^2.24.0",
     "moxios": "^0.4.0",
     "password-validator": "^5.0.3",
     "query-string": "^6.10.1",
     "react": "^16.12.0",
-    "@date-io/date-fns": "^2.2.0",
-    "@material-ui/pickers": "^3.2.10",
-    "enzyme-to-json": "^3.4.4",
-    "jest-canvas-mock": "^2.2.0",
-    "jest-localstorage-mock": "^2.4.0",
```
Comment (Jajabenit250) [inline package.json:96]: and some of these line too .

File: src/actions/signInAction.js [hunk 3]
```diff
@@ -3,20 +3,23 @@ export const USER_SIGNIN_SUCCESS = 'USER_SIGNIN_SUCCESS';
...
+export const signIn = (email, password) => async (dispatch) => {
     dispatch({ type: "LOADING", payload: true });
-    return await axios.post('https://blackninjas-backend-staging.herokuapp.com/api/v1/auth/signin', { "email" : email, "password" : password})
+    return await axios.post(`${process.env.BACKEND_BASE_URL}/api/v1/auth/signin`, { "email": email, "password": password })
         .then(response => {
             if (response.data.status === 200) {
-            const user = response.data.data;
-            localStorage.setItem('token', user);
-            dispatch({ type: "LOADING", payload: false });
-            dispatch(success(user));
```
Comment (Jajabenit250) [inline src/actions/signInAction.js:19]: this line don't have to be changed too.

**Review by Jajabenit250 — COMMENTED** [review]
Good Work so far, Kindly attend on the comment i left.

File: src/actions/signInAction.js (hunk 3 above)
Comment (VedaDom) [reply src/actions/signInAction.js:19]: You did not follow the eslintrc rules for indentation and you forgot to user .env variable.

File: package.json (hunk 2 above)
Comment (VedaDom) [reply package.json:96]: I'm going to collect this. I'm sorry it happened as a mistake.

**Review by mujohn26 — APPROVED** [review]

**Review by shemaeric1234 — APPROVED** [review]

**Review by shemaeric1234 — APPROVED** [review]

**Review by bdushimi — APPROVED** [review]

**Pull request merged**